OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -870,6 +882,18 @@ void Manager::initialize(SimTK::State& s, double dt )
 }
 //_____________________________________________________________________________
 /**
+* set and initialize a SimTK::TimeStepper
+*/
+void Manager::initializeTimeStepper(
+    const SimTK::System& sys, const SimTK::State& state)
```

Comment:
From my understanding, this is because of the inconsistency due to using the `Manager` for CMC as well (see [this line](https://github.com/opensim-org/opensim-core/pull/1386/files#diff-d43892ebaf9a963f51ea372ed57fcb78R746)).